sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_client.py

```diff
@@ -211,6 +212,15 @@ def _initialize(self, **kwargs) -> None:
             # try to get the key again next time unless we know we're forbidden to do so
             self._initialized = self._keys_get_forbidden
 
+    @distributed_trace
+    def create_rsa_private_key(self) -> KeyVaultRSAPrivateKey:
+        """Create an `RSAPrivateKey` implementation backed by this `CryptographyClient` and key.
```

Comment:
Should you document that it's actually a `KeyVaultRSAPrivateKey`, or is the base class fine here?

Reply:
The return type shows the more specific type, but you're right that it'd be best to be more specific in the docstring. Updating now.

Reply:
FWIW, .NET returns our specific implementation but, as a typed language, that probably makes more sense. We might want to hang properties off it that aren't declared by the base class. You're duck-typed, so I doubt it matters, but wanted to throw that out there anyway. Maybe it's worth still declaring the return type as your own so callers get better code completion, or does that work on duck-typed objects as well?